FILES.deploy-tools/root/playbooks/ndslabs-k8.yml

```diff
@@ -120,8 +120,16 @@
 - name: NDSLabs GLFS Cluster FileSystem Clients
   hosts: compute
   roles:
-    - { role: k8-glfs-client-set, when: "groups['glfs']|length > 0" }
+    - k8-glfs-client-set
```

Comment:
You've deleted a necessary clause here for the single node install.  GLFS clients are only deployed if GLFS nodes are configured (which they are not in the single-node case)